flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -139,6 +140,69 @@ public void shouldSubmitToJobClient() {
 	}
 
 	/**
+	 * Tests that invalid detached mode programs fail.
+	 */
+	@Test
+	public void testDetachedMode() throws Exception{
+		jobManagerSystem.actorOf(Props.create(SuccessReturningActor.class), JobManager.JOB_MANAGER_NAME());
+		Client out = new Client(config);
+
+		try {
+			PackagedProgram prg = new PackagedProgram(TestExecuteTwice.class, "/dev/random", "/tmp");
+			out.runDetached(prg, 1);
+		} catch (ProgramInvocationException e) {
+			assertEquals("The Job was submitted in detached mode. Only one execution is allowed. " +
```

Comment:
Can we make this message available somewhere such that it doesn't have to be copied over and over?